dkg/pedersen/handler.go

```diff
@@ -28,13 +28,18 @@ import (
 // Allows to exit the loop.
 const recvTimeout = time.Second * 30
 
-// the channel size used to stored buffered deals and responses. This arbitrary
-// value must be set according to the characteristic of the system: number of
-// nodes, networking, memory, etc...
-// const chanSize = 10000
-
 // constant used in the logs
 const newState = "new state"
+const badState = "bad state: %v"
+const failedState = "failed to switch state: %v"
+
+// enumeration of the node type
+const (
+	unknownNode = iota
```

Comment:
you can only declare `iota` once, expressions are implicitly repeated in a paren­thesized const declaration